core/src/main/java/com/google/cloud/sql/SqlAdminApiClientFactory.java

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright 2023 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.cloud.sql;
+
+import com.google.api.client.googleapis.javanet.GoogleNetHttpTransport;
+import com.google.api.client.http.HttpRequestInitializer;
+import com.google.api.client.http.HttpTransport;
+import com.google.api.client.json.JsonFactory;
+import com.google.api.client.json.gson.GsonFactory;
+import com.google.api.services.sqladmin.SQLAdmin;
+import java.io.IOException;
+import java.security.GeneralSecurityException;
+import java.util.Optional;
+import java.util.logging.Logger;
+
+public class SqlAdminApiClientFactory implements ApiClientFactory {
```

Comment:
Compare: SQLAdmin vs SqlAdmin here -- should we make this consistent with the generated library?

Reply:
I tried changing it but the lint checker wasn't happy.